app/experimenter/nimbus-ui/src/components/Summary/index.tsx

```diff
@@ -34,7 +34,7 @@ const Summary = ({ experiment }: SummaryProps) => {
 
       <div className="d-flex flex-row justify-content-between">
         <h2 className="h5 mb-3">Summary</h2>
-        {!status.draft && !status.review && (
+        {!status.draft && (
```

Comment:
Oh I didn't realize we had this check mirrored here, it would be nice if we did this in one place on the backend and just sent up a property to the frontend.

Reply:
Like an `apiLink` or something?

Reply:
Yeah a string that's generated on the backend and the frontend can just show it if it's not null?